generate_management_sdk.py

```
@@ -75,6 +93,146 @@ def preserve_custom_imports():
+        # Extract Kinde-specific exceptions
+        kinde_exceptions = []
+        lines = content.split('\n')
+        in_kinde_exceptions = False
+        for line in lines:
+            if line.strip() == "# Kinde-specific exceptions":
+                in_kinde_exceptions = True
+                kinde_exceptions.append(line)
+            elif in_kinde_exceptions and line.strip().startswith('class ') and 'Exception' in line:
+                kinde_exceptions.append(line)
+            elif in_kinde_exceptions and line.strip() == "pass":
+                kinde_exceptions.append(line)
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)